classes/list_music_album.rb

```diff
@@ -1,7 +1,12 @@
 class ListAlbum
     def self.list_album(albums)
         puts 'Your library is empty' if albums.empty?
-        albums.each_with_index do |album, index| puts album end
+        albums.each do |album| 
+            puts "Id: #{album['id']}),
+            class: #{album['class']},
+            On Spotify: #{album['on_spotify']},
+            Genre: #{album['genre']['name']}}"
+        end
     end
 end

```

Comment:
I think there isn't need for another class. We can add this as a method on the main.rb or app.rb file. 
![image](https://user-images.githubusercontent.com/62131896/153310326-a557be5b-f4f8-449f-be9a-f484428afd72.png)